cmd/blockstorage_list.go

```diff
@@ -0,0 +1,103 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/dustin/go-humanize"
+	"github.com/spf13/cobra"
+
+	"github.com/exoscale/cli/pkg/globalstate"
+	"github.com/exoscale/cli/pkg/output"
+	v3 "github.com/exoscale/egoscale/v3"
+)
+
+type blockstorageListItemOutput struct {
+	ID    v3.UUID                    `json:"id"`
+	Name  string                     `json:"name"`
+	Zone  v3.ZoneName                `json:"zone"`
+	Size  string                     `json:"size"`
+	State v3.BlockStorageVolumeState `json:"state"`
+}
+
+type blockstorageListOutput []blockstorageListItemOutput
+
+func (o *blockstorageListOutput) ToJSON()  { output.JSON(o) }
+func (o *blockstorageListOutput) ToText()  { output.Text(o) }
+func (o *blockstorageListOutput) ToTable() { output.Table(o) }
+
+type blockstorageListCmd struct {
+	cliCommandSettings `cli-cmd:"-"`
+
+	_ bool `cli-cmd:"list"`
+
+	Zone v3.ZoneName `cli-short:"z" cli-usage:"zone to filter results to"`
```

Comment:
```suggestion
	Zone v3.ZoneName `cli-short:"z" cli-usage:"zone to list"`
```

Reply:
https://github.com/exoscale/cli/blob/master/cmd/instance_list.go#L43
I wanted to stay consistent with other command :)